### DISCUSSION TITLE
Almost working for QFUS camera 9.0.35 FW

### DISCUSSION THREAD

**Discussion by 2blmaster** [roleoroleo/yi-hack-Allwinner-v2#730] [Q&A] [answered]
Hi!

I purchased a Yi 1080p Home Camera from Amazon Germany recently. I received a QFUS... model with 9.0.35... FW.
I nevertheless gave y211ga a try (supported FW 9.0.36 only) and first of all the hack installed sucessfully on the camera. (RTSP works, Web interface works, etc., nice!)

Nevertheless once I disable the App/Cloud features via the Web interface the Camera reboots constantly after few seconds, I asume it tries to connect "home" once Wireless connection is established and reboots once it fails...

Is this behaviour expected, and is there anything I can do? Am I the only one receiving this hardware/firmware combination so far? ;)

Thanks for your work so far and Cheers!
  2blmaster

**Comment by roleoroleo** [comment]
If the cam has  the fw 9.0.35, it's a y291ga and not a y211ga.
Please, use the correct version of the hack.

**Reply by 2blmaster** [reply]
Thanks for pointing that out. I was led more by the "Q" as first letter pointing towards the y211ga, instead of the 9.0.35 FW pointing towards the y291ga. It seems my camera is some kind of hybrid? ;)

Nevertheless I switched to the y291ga hack version, and the result is identical. Works well with app/cloud enabled, restarts periodically after few seconds with app/cloud disabled.

Or do I need to somehow wipe the y211ga hack from the camera first? I just changed the sd content from y211ga to y291ga...

Thanks!

**Reply by roleoroleo** [reply]
Did you enable swap file?

**Reply by 2blmaster** [reply]
Yes, swap file is enabled.
Both Hack Versions (y211ga and y291ga) work well (tested web interface and RTSP stream only) as long as the "Disable cloud" switch is not active.
Reproducably and as soon as I tick "Disable Cloud" and reboot for the change to take effect, the camera constantly reboots few seconds after the blue light became stable...

**Reply by 2blmaster** [reply]
Any logs that I could check/provide? Time in between reboots could almost be enough to SSH onto the camera and run a few commands ;)

**Reply by 2blmaster** [reply]
Interestingly I found another configuration combination that works without constant rebooting:
achtivate the "Disable cloud" feature, AND deactivate "Swap File" (AND deactivate "Snapshot" as per recommendation in the FAQ)
This is for me an acceptable working state, as I do not need Snapshots, most likely as a consequence do not need Swap, and freed the camera from the app/cloud.
Nevertheless I am wondering how these two optins work together, because as previously described the camera also works well when "Swap file" is activated, AND "Disable Cloud" is deactivated.
Maybe it has something to do with my cheap and crappy no-name SD-Card?

Nevertheless I can confirm that under the above conditions the y291ga hack Version 0.2.9 works with a YI 1080p Home Camera, "EU" Version, FW: 9.0.35..., DID: QFUSY..., S/N: QY291GEUW... (and differing from the "RFUS" from your compatibility list)

Cheers!
![IMG_20231024_212827](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/148819847/d22957cd-9540-4959-abd1-13992a3bcf0a)
![IMG_20231024_212855](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/148819847/33388e9e-409a-4c70-831d-2e9014d2659a)

**Reply by 2blmaster** [reply, marked as answer]
Indeed switching to a well-known brand SD card seems to solve the issue. I can have "Disable Cloud" activated, as well as "Snapshot" enabled and "Swap File" enabled.

For me the issue is solved this way.

Any insights on why the camera would reboot over and over on a crappy SD card, but only when Swap File and Disable Cloud are *both* active?

**Reply by roleoroleo** [reply]
It's probably a race condition in the start sequence.
But it's nor simple to find it.
